superset/databases/schemas.py

```diff
@@ -552,6 +552,25 @@ class DatabaseFunctionNamesResponse(Schema):
 
 
 class ImportV1DatabaseExtraSchema(Schema):
+    # pylint: disable=no-self-use, unused-argument
+    @pre_load
+    def fix_schemas_allowed_for_csv_upload(
+        self, data: Dict[str, Any], **kwargs: Any
+    ) -> Dict[str, Any]:
+        """
+        Fix ``schemas_allowed_for_csv_upload`` being a string.
```

Comment:
would it make sense to run a migration/script to update the data in the table directly? I assume the size of these tables is relatively small.

Reply:
For sure! But I'm planning to work on a separate PR that fixes `schemas_allowed_for_csv_upload` being stored incorrectly, I thought of doing the migration there.

Reply:
are we fixing the fe bug separately?

Reply:
Yep! :)